### PR TITLE
Added sky and C/N0 vs time plots.

### PR DESCRIPTION
**Pull request by adamshapiro0**
# New Features
- Added GNSS sky plot
- Added GNSS C/N0 vs time plot
- Allow wildcards in `--plot` specifiers to show multiple plots

### CODE REVIEW

File: python/fusion_engine_client/analysis/analyzer.py [hunk 1]
```diff
@@ -742,8 +743,163 @@ def _plot_data(name, idx, marker_style=None):
...
+            rows=1, cols=1,  print_grid=False, shared_xaxes=True,
+            subplot_titles=['C/N0 (L1 Only)'])
+
+        figure['layout'].update(showlegend=True, modebar_add=['v1hovermode'])
+        figure['layout']['xaxis1'].update(title="Time (sec)", showticklabels=True)
+        figure['layout']['yaxis1'].update(title="C/N0 (dB-Hz)")
+
+        # Assign colors to each satellite.
+        data_by_sv = GNSSSatelliteMessage.group_by_sv(data)
+        svs = sorted(list(data_by_sv.keys()))
```
Comment (wbrannon) [inline python/fusion_engine_client/analysis/analyzer.py:862]: Does this need to be a sorted list?

File: python/fusion_engine_client/utils/argument_parser.py [hunk 2]
```diff
@@ -151,7 +152,12 @@ def _format(option):
                 elif '=' in option:
                     return None
                 else:
-                    return f"{option}[=<{', '.join(TriStateBooleanAction.POSSIBLE_VALUES)}>]"
+                    nonlocal options_printed
```
Comment (wbrannon) [inline python/fusion_engine_client/utils/argument_parser.py:155]: Huh I've never seen this `nonlocal` keyword before - pretty neat.

File: python/setup.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
             'argparse-formatter>=1.4',
             'colorama>=0.4.6',
```
Comment (wbrannon) [inline python/setup.py:14]: Make sure to rebase your other PR where you change the `colorama` version here

**Review by wbrannon — APPROVED** [review]
Looks good.

File: python/fusion_engine_client/analysis/analyzer.py (hunk 1 above)
Comment (adamshapiro0) [reply python/fusion_engine_client/analysis/analyzer.py:862]: Not right here technically, but I want to loop over it in order below so the traces on the plot are listed in order. I could sort it there but figured better to do it here just so `svs` is in order if we want to look at it for debugging.

File: python/fusion_engine_client/utils/argument_parser.py (hunk 2 above)
Comment (adamshapiro0) [reply python/fusion_engine_client/utils/argument_parser.py:155]: Yeah it's not a super common thing, and they added it in Python 3. It lets you modify a primitive variable that lives outside the scope of a nested function. Before that in Python 2 you had to do ugly stuff to make your primitive into an object to work around the limitation. Something like:
```py
my_number = [3]
def my_func():
    my_number[0] += 5
```

File: python/setup.py (hunk 3 above)
Comment (adamshapiro0) [reply python/setup.py:14]: Yep thanks for the reminder

**Pull request merged**